portlist/portlist_linux.go

```diff
@@ -26,8 +26,8 @@ const pollInterval = 1 * time.Second
 
 // TODO(apenwarr): Include IPv6 ports eventually.
 // Right now we don't route IPv6 anyway so it's better to exclude them.
-var sockfiles = []string{"/proc/net/tcp", "/proc/net/udp"}
-var protos = []string{"tcp", "udp"}
+var sockfiles = []string{"/proc/net/tcp", "/proc/net/tcp6", "/proc/net/udp", "/proc/net/udp6"}
+var protos = []string{"tcp", "tcp", "udp", "udp"}
```

Comment:
not sure how useful this variable is since it's just `strings.TrimSuffix(filepath.Base(fname), "6")`